sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/JsonSerializable.java

```diff
@@ -17,6 +17,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import io.micrometer.core.lang.Nullable;
```

Comment:
bug: I don't think you need Nullable from this lib.

Reply:
@moderakh  - Though everywhere in the code, we have this annotation (from micrometer). But I agree, we should replace this with `@reactor.util.annotation.Nullable` as there is no need to depend on micrometer for `Nullable` annotation

Reply:
yes, we can use micrometer or reactor, both have same effect. Or better, we could use `javax.annotation.Nonnull`, (which is used by micrometer or reactor Nullable annotations).

Reply:
My concern with `javax.annotation.Nonnull`  -> I recently tested our SDK with Java 11 and this was giving issues as there is no module for javax, so I had to change them to reactor Nullable annotation.
Though java 11 is far in the future, but something worth considering.

Reply:
thanks, Kushagra for sharing this, sounds good, will use Reactor NonNull

Reply:
done in the lastest [commit](https://github.com/Azure/azure-sdk-for-java/pull/9111/commits/b972e8c088bb80842e7fa0633740f49128f1991b).